mode/vue/index.html

```diff
@@ -0,0 +1,88 @@
+<!doctype html>
```

Comment:
So the examples of single-file vue apps I found online seem to use `<script>` tags to delimit parts of the document. Since the `htmlmixed` mode already allows you to customize the way the content of different types of script tags get parsed, I'm not convinced that this mode adds a lot.